consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/validator/ForkingValidatorProvider.java

```diff
@@ -33,13 +32,13 @@
 public class ForkingValidatorProvider implements ValidatorProvider {
 
   private final Blockchain blockchain;
-  private final ValidatorSelectorForksSchedule forksSchedule;
+  private final BftForksSchedule<QbftConfigOptions> forksSchedule;
```

Comment:
The type parameter now couples Qbft to ForkingValidatorProvider...is that a problem or is it this class likely to remain in the qbft package?